app/constants/urls.constants.ts

```diff
@@ -1,2 +1,5 @@
 export const checkUsername = (URL: string, username: string) =>
   `${URL}/users/usernameCheck/${username}`;
```

Comment:
can we change `URL` to `HOST` ?

Reply:
@isVivek99, Well this code was written by you only so I don't know if you are asking or telling here 😄

Reply:
i'm asking :),

Reply:
@isVivek99 okay i'll update it. Better to be consistent